src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/FindBreakpointEvidenceSpark.java

```diff
@@ -41,19 +41,29 @@
 import static org.broadinstitute.hellbender.tools.spark.sv.evidence.BreakpointEvidence.ReadEvidence;
 
 /**
- * Tool to discover genomic breakpoints associated with structural variants.
+ * (Internal) Produces local assemblies of genomic regions that may harbor structural variants
```

Comment:
I'm not keen on labeling these standalone tools as internal nor only of interest to developers. I think it is likely more the case that folks will start with these standalone tools before they progress to using Spark. There is some overhead to set up to run Spark, and we do not have clear instructions towards this as of yet. The future may be here but it is not evenly distributed.

Reply:
@sooheelee They aren't standalones; they still rely on Spark.  It's just that they do a small fraction of the useful work--pieces that we needed to do repeatedly during development.  I do think they are internal and useless to everyone but developers, but it doesn't really matter all that much as long as we don't have to fiddle with documenting them any further.